dist/highcharts-layout/js/message.js

```diff
@@ -3,20 +3,19 @@ const hcUtils = {
     parseOptions: function (chartOptions) {
         const parseFunction = this.parseFunction;
 
-        var options = JSON.parse(chartOptions, function (val, key) {
+        let options = JSON.parse(chartOptions, function (val, key) {

```

Comment:
Why not `const`? Here and in the other places.

Reply:
Great, it makes total sense I forgot to think that in this context we are not assigning this variable again.

Reply:
Thank you for the suggestion, will fix also.